WcaOnRails/db/migrate/20180629112054_add_color_to_venue_room.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class AddColorToVenueRoom < ActiveRecord::Migration[5.2]
+  def change
+    add_column :venue_rooms, :color, :string, default: VenueRoom::DEFAULT_ROOM_COLOR, allow_blank: false, limit: 7
```

Comment:
What does this `allow_blank` option do? Are you sure you don't mean `null: false` here instead?

I'm not in love with the default being here. I'm concerned that if someone ever changes `DEFAULT_ROOM_COLOR` they won't realize a database migration is needed. Maybe you could instead allow for room's colors to be unspecified, and then the ruby/javascript code could explicitly handle that by falling back to a particular color?

Reply:
I totally meant `null: false`...

Right I'll add an update to all rooms without color to the migration :)
For the new rooms it's already handled through a `before_validation`.